File: Haskell/0001.hs
-- Define the main function
main :: IO ()
main = do
    -- Calculate the sum of multiples of 3 or 5 below 1000
    let sumMultiples = sum [x | x <- [1..999], x `mod` 3 == 0 || x `mod` 5 == 0]
    -- Print the result
    print sumMultiples

File: Perl/0001.pl
use strict;
use warnings;

# Function to calculate the sum of multiples of 3 or 5 below a given limit
sub sum_of_multiples {
    my ($limit) = @_;
    my $sum = 0;

    for (my $i = 1; $i < $limit; $i++) {
        if ($i % 3 == 0 || $i % 5 == 0) {
            $sum += $i;
        }
    }

    return $sum;
}

# Define the limit
my $limit = 1000;

# Calculate the result
my $result = sum_of_multiples($limit);

# Print the result
print "The sum of all multiples of 3 or 5 below $limit is: $result\n";